akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -101,6 +101,12 @@ object Recovery {
    */
   val none: Recovery = Recovery(toSequenceNr = 0L, fromSnapshot = SnapshotSelectionCriteria.None)
 
+  /**
+   * Don't load snapshot and replay only last event.
+   */
+  val replayOnlyLast: Recovery =
+    Recovery(fromSnapshot = SnapshotSelectionCriteria.NoSnapshotAndReplayOnlyLast)
```

Comment:
It's difficult to change the classic `Recovery` class since that is a case class, so here it is captured with a SnapshotSelectionCriteria and in the end we can turn that into fromSequenceNr = -1

Reply:
this was not needed